src/components/banner/banner.js

```diff
@@ -1,32 +1,59 @@
 import React from 'react';
-import banner from './banner.svg';
 import PropTypes from 'prop-types';
-
 import { withStyles } from '@material-ui/core/styles';
 
+import Wrenches from './wrenches';
+
 const styles = theme => ({
 	container: {
-		position: 'relative'
+		backgroundColor: '#352A3A'
+	},
+	content: {
+		display: 'flex',
+		position: 'relative',
+		alignItems: 'center',
+		justifyContent: 'left',
+		maxWidth: '800px',
+		height: '400px',
+		margin: 'auto',
```

Comment:
I see what you are trying to do here. Maybe we can use flexbox to solve the problem.
We use a flex and use `justifyContent: 'space-between'`

Reply:
I couldn't use flexbox here because I needed the items to overlap as the screen width shrinks. So the wrenches basically sit in a fixed location in a divider while the words have absolute positioning relative to that divider.

Reply:
Ohhhhhhh, I see what you mean now. I tried it out myself and yes, it is a limitation of flexbox. Then we will keep your approach. very smart la